master/setup.py

```diff
@@ -450,6 +450,8 @@ def define_plugin_entries(groups):
         'txgithub',
         'ramlfications',
         'mock',
+        'shutilwhich',
```

Comment:
there seem no condition on the Python version here (unlike the place where it's used)